00-programming-fundamentals/exercises/merge-conflict.md

```diff
@@ -55,6 +55,9 @@ Because each member of your group now has a different version of the recipe file
     - How could this process have been made easier?
     - Write up a set of steps to follow when dealing with merge conflicts.
 
+##### Activity Workflow
+![activity workflow](images/mergeConflictDiagram.png)
+
```

Comment:
Can you add the link to the original editable diagram here so if changes need to be made later we know where to look?